src/oncall/ui/static/js/oncall.js

```diff
@@ -2021,6 +2022,23 @@ var oncall = {
       }
     },
     renderPage: function(data){
+      $.when(
+        oncall.data.modesPromise
+      ).done(function() {
+        let contactModes = [];
+
+        for(let key in data.contacts)
+        {
+          let currentMode = oncall.data.modes.find(x => x.name === key);
+          contactModes.push({
+            label: currentMode.label,
```

Comment:
Here, why not capitalize the mode name instead of dealing with the schema change and labels?